website/docs/proof-system/proof-system.md

```diff
@@ -55,7 +55,7 @@ In addition to the links in the sidebar, we recommend the following resources:
 [ReceiptClaim]: https://docs.rs/risc0-zkvm/*/risc0_zkvm/struct.ReceiptClaim.html
 [SegmentReceipts]: https://docs.rs/risc0-zkvm/*/risc0_zkvm/struct.SegmentReceipts.html
 [SegmentReceipt]: https://docs.rs/risc0-zkvm/*/risc0_zkvm/struct.SegmentReceipt.html
-[flat receipt]: https://docs.rs/risc0-zkvm/0.19.1/risc0_zkvm/struct.FlatReceipt.html
+[flat receipt]: https://docs.rs/risc0-zkvm/latest/risc0_zkvm/enum.InnerReceipt.html#variant.Flat
```

Comment:
```suggestion
[flat receipt]: https://docs.rs/risc0-zkvm/*/risc0_zkvm/enum.InnerReceipt.html#variant.Flat
```

BTW this is going to break very soon, we just renamed this in `0.20`. I think we should consider trying to avoid pointing to code from these sorts of docs.

Reply:
Hmm...  I know the naming is unstable but not sure how this is solved by avoiding the pointer.

Can you clarify what you mean by "these sorts of docs" and what you have in mind as an alternative?  We point to the crate docs on a large number of pages, and providing a route for users to dig deeper into the details seems like a primary purpose of these docs.

Reply:
Like, I guess on most pages we could point to `terminology.md` rather than the crate docs, but it strikes me as a big mistake to avoid navigation paths from dev docs website to crate docs.

Reply:
will revise from `latest` to `0.19.1` for now, as I believe our convention is to avoid `latest` in order to make sure that we have an effective link audit as we push new releases

Reply:
Consistency, it's shorter.

But I'm coming around to the idea that for `/docs` we should always point to the latest stable. But now we need to have a process where when we cut a new release we have to comb thru the `/docs` to update all the versions and do a pass on naming changes.

Reply:
Hmm...I'm happy to defer to your judgment on this. 

The site right now includes use of `/0.18/`, `/0.19/` more than `/latest/`. I'm having trouble figuring out if/where we use `/*/` because ctrl-f isn't place nice with `*`.